test/unit_tests/aie/02_lock_acquire_release/test.cpp

```diff
@@ -26,7 +26,9 @@ int main(int argc, char *argv[]) {
   printf("test start.\n");
 
   aie_libxaie_ctx_t *_xaie = mlir_aie_init_libxaie();
+  printf("1\n");
   mlir_aie_init_device(_xaie);
+  printf("2\n");
```

Comment:
remove `printf`

Reply:
Thanks, addressed in [f18a8d6](https://github.com/Xilinx/mlir-aie/pull/570/commits/f18a8d66b255ed857b56380a62fe27e5ba046e25).